cockatrice/src/settingscache.h

```diff
@@ -194,6 +196,7 @@ public slots:
     void setLeftJustified( const int _leftJustified);
     void setMasterVolume(const int _masterVolume);
     void setCardInfoViewMode(const int _viewMode);
+    void setHighlightWords(const QString _highlightWords);
```

Comment:
Use `const QString &_highlightWords` instead here. This will prevent making a copy.